dir.props

```diff
@@ -4,6 +4,10 @@
 
   <Import Project="dir.common.props" />
 
+  <PropertyGroup>
+    <LangVersion>8.0</LangVersion>
```

Comment:
Had to move the `LangVersion` definition to here and `tests\dir.props`, since those are the only things that actually import `$(RoslynPropsFile)`.

I think it would be nice to get this cleaned up (and using `Directory.Build.props`, etc). But that is a more complicated change and should be done separately. Doing so would allow this logic to be centrally managed, rather than duplicated in multiple places.

Reply:
Right, that would be part of "Arcade migration".